src/game/game.js

```diff
@@ -413,7 +413,7 @@
             });
 
             if(!runCodeCache[userId].globals._) {
-                runCodeCache[userId].globals._ = _.runInContext();
+                runCodeCache[userId].globals._ = _.runInContext(runCodeCache[userId].globals);
```

Comment:
This doesn't seem to work as intended. `runCodeCache[userId].globals)` contains only `RawMemory` and `console`, and no global prototypes, so it still uses host prototypes in lodash functions. Do you have any tests confirming that this change makes any sense?